spec/presenters/firm_profile_presenter_spec.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.describe FirmProfilePresenter do
+  subject do
+    described_class.new(firm_result)
+  end
+
+  let(:firm_result) { double(FirmResult) }
+
+  describe '#type_of_advice_method' do
+    context 'face to face available' do
+      it do
```

Comment:
I was expecting the text within the `it .. do` to explain the desired outcome for this spec.